server/app.py

```diff
@@ -33,6 +33,15 @@ def create_game():
     return {'game_id': game_id}
 
 
+@app.route('/blackjack/api/v1/games/<game_id>', methods=['POST'])
```

Comment:
The POST verb is usually for sending data up to the server, with a post body. The HTTP method you probably want is GET.